central/debug/service/service.go

```diff
@@ -60,8 +60,8 @@ const (
 
 	centralClusterPrefix = "_central-cluster"
 
-	metricsPullTimeout     = 10 * time.Second
-	diagnosticsPullTimeout = 10 * time.Second
+	metricsPullTimeout     = 20 * time.Second
+	diagnosticsPullTimeout = 20 * time.Second
```

Comment:
it seems like this could be dynamic based on ctx timeout passed via roxctl or the ui

Reply:
Issue created https://issues.redhat.com/browse/ROX-12940

Reply:
And another https://issues.redhat.com/browse/ROX-12939